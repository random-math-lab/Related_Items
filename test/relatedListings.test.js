import React from 'react';
import { shallow } from 'enzyme';
import RelatedListings from '../client/src/components/relatedListings.jsx';


describe('relatedListings', () => {
    const sampleData = [
        {
            "id": 18,
            "type": "ENTIRE HOUSE",
            "title": "saepe et doloremque",
            "city": "New Eddieview",
            "price": 120,
            "rating": 4,
            "votes": 398,
            "img": "https://relatedplaces.s3-us-west-1.amazonaws.com/18.jpg"
        },
        {
            "id": 21,
            "type": "ENTIRE LOEFT",
            "title": "quisquam ut odio",
            "city": "Port Karli",
            "price": 103,
            "rating": 3,
            "votes": 705,
            "img": "https://relatedplaces.s3-us-west-1.amazonaws.com/21.jpg"
        },
        {
            "id": 31,
            "type": "ENTIRE HOUSE",
            "title": "velit voluptatem nesciunt",
            "city": "Weberside",
            "price": 98,
            "rating": 4,
            "votes": 545,
            "img": "https://relatedplaces.s3-us-west-1.amazonaws.com/31.jpg"
        }
    ]
    const component = shallow(<RelatedListings listings={sampleData} pos={0}/>);
    
    it('should render 3 places', () => {
        expect(component.children().length).toBe(3);
    })
    
  it('should render RelatedListings', () => {
  
    expect(component).toMatchSnapshot();
  })


})