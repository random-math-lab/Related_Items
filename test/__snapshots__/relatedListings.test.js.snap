// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`relatedListings should render RelatedListings 1`] = `ShallowWrapper {}`;
